apps/passport-client/src/dispatch.ts

```diff
@@ -1248,3 +1254,19 @@ async function removeAllPCDsInFolder(
   update({ pcds: state.pcds });
   window.scrollTo({ top: 0 });
 }
+
+async function initializeStrich(
+  state: AppState,
+  update: ZuUpdate
+): Promise<void> {
+  if (!appConfig.strichLicenseKey) {
+    console.log("Strich license key is not defined");
+    return;
+  }
+  try {
+    await StrichSDK.initialize(appConfig.strichLicenseKey);
+    update({ strichSDKstate: "initialized" });
```

Comment:
i wonder if we should do a `promise.race` here in case it takes more than 10 seconds to initialize

Reply:
Done!